templates/status.html

```diff
@@ -44,7 +44,7 @@
                 else if (!res['possible'])
                     $("#" + site)[0].innerHTML = 'Impossible';
                 else if (res['status'] == '404' || res['status'] == '301')
-                    $("#" + site)[0].innerHTML = 'Available';
+                    $("#" + site)[0].innerHTML = `<a href="'+signup+'">Available</a>`;
```

Comment:
I encourage you to use template strings here, as it makes it a bit cleaner :)
Instead of using `+` to concatenate, you can use `${signup}`

Reply:
Using this makes Travis CI builds to fail.

Reply:
@siddhpant it was because of wrong indentation not because you've used `${signup}`.

Reply:
Maybe a space got misplaced :frowning_face:

Reply:
@seeeturtle Lol it's the same indentation in my text editor and in Github online editor, but diff is hsowing it different!

Reply:
Finally fixed.